Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt

```diff
@@ -316,8 +320,9 @@ private fun LessonsSheet(
 ) {
     // Use the fraction that the sheet is open to drive the transformation from FAB -> Sheet
     val fabSize = with(DensityAmbient.current) { FabSize.toPx() }
+    val fabHeight = fabSize + InsetsAmbient.current.systemBars.bottom
```

Comment:
Nit: Technically the insets aren't part of the FAB height.

Reply:
Done.